content/news/050/index.md

```diff
@@ -203,20 +200,20 @@ _Discussions: ([/r/roguelikes][monk-tower-reddit])_
 
 ![You are Merlin screenshot](you-are-merlin.png)
 
-You are Merlin ([Web Game][you-are-merlin-web], [GitHub - Rust/CLI][you-are-merlin-github], [GitHub - WASM][you-are-merlin-www-github]) by @hseager
+You are Merlin ([Web Game][you-are-merlin-web], [GitHub - Rust/CLI][you-are-merlin-github], [GitHub - WASM][you-are-merlin-www-github]) by [@hseager](https://github.com/hseager)
 is a text adventure game that compiles to both CLI and WebAssembly.
 
 This initial version features a main quest, side quests, items and a boss fight.
-Players can choose their favourite theme such as Zelda, Warcraft and Fallout. 
-With the web version, players can select their prefered input method of buttons or keyboard and fully works on mobile.
+Players can choose their favourite visual theme such as Zelda, Warcraft and Fallout. There is also mobile support.
```

Comment:
```suggestion
Players can choose their favourite visual theme such as Zelda, Warcraft, and Fallout. There is also mobile and web support.
```

Reply:
Misleading: there is only CLI and web. The web version supports mobile devices.

Reply:
Counteroffer: [`5fa2fed` (#1496)](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1496/commits/5fa2fed7dfbe48aee35e083387b3ba4a6a10fa8e)

Reply:
Looks good, but can we also add the comma after "Warcraft"?

Reply:
Actually I added a few others as well, should be easier to merge from a PR https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1503